runtime/src/main/java/org/corfudb/runtime/collections/TransactionPoller.java

```diff
@@ -69,18 +82,12 @@ private void pollTxStream() {
  
         long lastReadAddress = streamContexts.get(0).getLastReadAddress();
 
-        StreamOptions options = StreamOptions.builder()
-                .ignoreTrimmed(true)
-                .cacheEntries(false)
-                .build();
-        IStreamView txStream = runtime.getStreamsView()
-                .get(ObjectsView.TRANSACTION_STREAM_ID, options);
         log.trace("Seeking txStream to {}", lastReadAddress + 1);
-        txStream.seek(lastReadAddress + 1);
+        txnStream.seek(lastReadAddress + 1);
         log.trace("txStream current global position after seeking {}, hasNext {}",
-                txStream.getCurrentGlobalPosition(), txStream.hasNext());
+                txnStream.getCurrentGlobalPosition(), txnStream.hasNext());
 
-        List<ILogData> updates = txStream.remaining();
+        List<ILogData> updates = txnStream.remaining();
```

Comment:
This call can throw ExecutionError due to Guava leakage we have seen in some testing.  ExecutionError thrown by google.localcache library. It can throw InterruptedException too if it gets interrupted.

Reply:
`pollTxStream` is called by `run` which has a try/catch block around it.

Reply:
For ExecutionError thrown by Guava, it is kind of a Guava bug, we should just ignore it. The current implementation propagate any throwable to each subscriber' contexts which will block them polling again in future.